cypress/e2e/example.cy.js

```diff
@@ -1,51 +1,68 @@
-import { faker } from '@faker-js/faker';
+
 /// <reference types='cypress' />
 
 describe('Bank app', () => {
-  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
-  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
-  const balance = depositAmount - withdrawAmount;
-  const user = 'Harry Potter';
-  const accountNumber = '1004';
-
-  before(() => {
-    cy.visit('/');
-  });
+  let balance;
+  const fullName = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const currency = 'Dollar';
+
+  const depositAmount = '200';
+  const withdrawAmount = '100';
+
+  const depositSuccessMessage = 'Deposit Successful';
+  const withdrawlSuccessMessage = 'Transaction successful';
```

Comment:
There is a typo in the variable name `withdrawlSuccessMessage`. It should be `withdrawalSuccessMessage` to match the correct spelling of 'withdrawal'.